index.html

```diff
@@ -1,497 +1,212 @@
-                            <button type="button" class="btn" id="next"onclick="nextTweet()">
-                                Next Tweet! &nbsp; →
-                            </button>
-                        </div>
-                    </div>
-                </div>
-                <!-- Stop ignoring -->
-            </section>
-            <!-- End of Loklak API -->
-            <!-- Mentors Section -->
-            <section class="mentors" id="mentors" href="#mentors">
-                <div class="row mentors-wrapper">
-                    <!-- Start ignoring BootLintBear -->
-                    <h2>MENTORS</h2>
-                    <!-- Stop ignoring -->
-                    {% for mentor in site.data.mentors %}
-                    <div class="col-xs-6 col-sm-4 col-md-3 col-lg-2">
-                        <div class="card">
-                            <div class="avatar">
-                                {% if mentor.image %}
-                                <img src="images/mentors/{{mentor.image}}" alt="{{ mentor.name }}">
-                                {% else %}
-                                <img src="images/mentors/member.png" alt="{{ mentor.name }}">
-                                {% endif %}
-                                <div class="social">
-                                    Show more<br>
-                                    information
-                                </div>
-                                <p style="display: inline-block;">{{mentor.name}}</p>
-                            </div>
-                        </div>
-                        {% include popup.html person=mentor who='mentor' %}
-                    </div>
-                    {% endfor %}
-                </div>
-            </section>
-        </div>
-            <!-- End of Mentor Section -->
-        <div class="container-fluid contribthank">
-            <div id="contributors-thanks" href="#contributors">
-                <div id="contributors-thanks-list" class="row">
-                    <div class="col-xs-12">
-                        <div class="content">
-                            <div class="visible">
-                                <p id="name">
-                                    Thanks for your contributions, everyone!
-                                </p>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
+    <link rel="stylesheet" href="ism/css/my-slider.css" media="screen">
 
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.2/html5shiv.js"></script>
+      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+    <style>
+    #map {
```

Comment:
Once this is ready to merge I'll make it in separate file, as of now I don't want merge conflicts in a lot of files so. Also, I'm checking original file for reference so don't wanna remove its css. But I'll do it once ready to merge.